containers/DvmState.ts

```diff
@@ -81,15 +83,15 @@ const useContractState = () => {
   // get state on setting of contract
   useEffect(() => {
     queryState();
-  }, [dvm, priceIdentifier, expirationTimestamp]);
+  }, [votingContract, priceIdentifier, expirationTimestamp]);
 
   // get state on each block
   useEffect(() => {
-    if (block$ && dvm) {
+    if (block$ && votingContract) {
```

Comment:
Also maybe add `storeContract` here for consistency

Reply:
Actually, I'm not sure you even need `votingContract` here, maybe you only need `block$`

Reply:
Agreed.

Reply:
cool. I changed it to. PTAL